Sources/CameraControlARView/ARViewContainer.swift

```diff
@@ -5,27 +5,40 @@
 //  Created by Joseph Heck on 2/9/22.
 //
 
+#if os(iOS)
+import UIKit
+#elseif os(macOS)
 import Cocoa
+typealias UIViewRepresentable = NSViewRepresentable
+#endif
 import RealityKit
 import SwiftUI
 
 /// A SwiftUI representable view that wraps an underlying augmented reality view with camera controls instance.
 ///
 /// Create an ``CameraControlARView`` externally and hand it into the container so that you can interact with the
 /// view controls, or the underlying scene, from within SwiftUI.
-public struct ARViewContainer: NSViewRepresentable {
+public struct ARViewContainer: UIViewRepresentable {
     /// The type of view this container wraps.
     public typealias NSViewType = RealityKit.ARView
 
     /// The wrapped ARView with camera controls enabled.
-    public var cameraARView: CameraControlARView
+    public var arView: CameraControlARView
```

Comment:
Why rename this to arView? I was naming it explicitly different (cameraArView) to make the initialization code below (in `makeNSView`) more explicit about what was being set to the externally visible "arView" property.

If there's a stunningly good reason to rename this, please update the documentation reference to use the correct name in `Sources/CameraControlARView/Documentation.docc/Extensions/ARViewContainer.md` (you can check by building the documentation locally with `<shift-control-command-D>` and verifying there aren't any warnings.